Orange/misc/distmatrix.py

```diff
@@ -101,6 +101,8 @@ def invert(self, typ):
         elif typ == 1:
             return 1.-self
         elif typ == 2:
+            return np.max(self) - self
+        elif typ == 3:
             return 1./self
```

Comment:
So I looked through the codebase, and this method is used exactly ... nowhere.

[YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)